tests/main/helpers/test_direct_award_helpers.py

```diff
@@ -0,0 +1,21 @@
+from app.main.helpers.direct_award_helpers import set_direct_award_display_status
+import pytest
+
+
+@pytest.mark.parametrize(
+    ('result', 'expected'),
+    [
+        ('awarded', 'Awarded'),
+        ('cancelled', 'The work has been cancelled'),
+        ('none-suitable', 'No suitable services found')
+    ]
+)
+def test_status_displays_correctly(result, expected):
+    framework = {'family': 'g-cloud'}
+    project = {'outcome': {'result': result}}
+    status = set_direct_award_display_status(project, framework)
+    assert status == expected
+
+# TODO test for link statuses - I get the following error:
+# Attempted to generate a URL without the application context being pushed.
+# This has to be executed when application context is available.
```

Comment:
The `url_for` function needs the Flask app context to resolve the URLs. To test, you can either provide the app context (there's an example of doing this in a test [here](https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/master/tests/main/views/test_marketplace.py#L1582) - that test needs access the Flask app config but it's the same principle), or you can patch out the `url_for` function to return whatever you like. I'd recommend the former 😸